### PR TITLE
Initialize Blazegraph namespaces on service boot

### PR DESCRIPTION
**Pull request by umbreak**
Related to the issue: https://github.com/BlueBrain/nexus-kg/issues/13

### CODE REVIEW

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by bogdanromanx** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

File: .jvmopts [hunk 1]
```diff
@@ -1,2 +1,10 @@
+-Dfile.encoding=UTF8
 -Xms2G
 -Xmx6G
+-XX:ReservedCodeCacheSize=250M
```
Comment (bogdanromanx) [inline .jvmopts]: set the value to 350M and add an additional entry:
`-XX:MaxMetaspaceSize=2G`

**Pull request merged**